Up versions and add changelog. (#355)
heremaps/here-data-sdk-typescript: @here/olp-sdk-dataservice-read/README.md

                    settings: olpClientSettings
            });
            /**

>              * Get some data from the layer by ID

Reviewer: ```suggestion
             * Gets data from a layer using its ID.
```
